test/simulation/xudtest/node.go

```diff
@@ -253,6 +253,18 @@ func (hn *HarnessNode) ConnectRPC(useMacs bool) (*grpc.ClientConn, error) {
 	}
 
 	tlsCreds, err := credentials.NewClientTLSFromFile(hn.Cfg.TLSCertPath, "")
+	var tries = 3
+	for tries > 0 {
+		if err != nil {
+			fmt.Printf("failed to create TLS from file with error: %v, remaining tries: %v\n", err, tries-1)
+			time.Sleep(3 * time.Second)
```

Comment:
3 seconds seems like kind of a long time between tries, would it make sense to have more tries with just a 1 second wait between tries?